src/services/boardService.test.js

```diff
@@ -0,0 +1,38 @@
+const board = require('./boardService');
```

Comment:
keep using imports - be consistent across all files. Also, name the import what you're importing. You are importing the boardService here - not the board.